config/webpack.config.web2.js

```diff
@@ -0,0 +1,255 @@
+      ],
+    },
+    devServer: {
+      inline: false,
+      before: (app /*: ExpressApp */) => {
+        const apiRoot = "/api/v1/data";
+        const rejectCache = (_unused_req, res /*: ExpressResponse */) => {
+          res.status(400).send("Bad Request: Cache unavailable at runtime\n");
+        };
+        app.get(`${apiRoot}/cache`, rejectCache);
+        app.get(`${apiRoot}/cache/*`, rejectCache);
+        app.use(
+          apiRoot,
+          express.static(
+            process.env.SOURCECRED_DIRECTORY ||
+              path.join(os.tmpdir(), "sourcecred")
+          )
+        );
+      },
+    },
+    output: {
+      // The build folder.
+      path: paths.appBuild2,
+      // Generated JS file names (with nested folders).
+      // There will be one main bundle, and one file per asynchronous chunk.
+      // We don't currently advertise code splitting but Webpack supports it.
+      filename: "static/js/[name].[chunkhash:8].js",
+      chunkFilename: "static/js/[name].[chunkhash:8].chunk.js",
+      // Point sourcemap entries to original disk location (format as URL on Windows)
+      devtoolModuleFilenameTemplate: (
+        info /*:
+        {|
+          // https://webpack.js.org/configuration/output/#output-devtoolmodulefilenametemplate
+          +absoluteResourcePath: string,
+          +allLoaders: string,
+          +hash: string,
+          +id: string,
+          +loaders: string,
+          +resource: string,
+          +resourcePath: string,
+          +namespace: string,
+        |}
+        */
+      ) =>
+        path
+          .relative(paths.appSrc, info.absoluteResourcePath)
+          .replace(/\\/g, "/"),
+      // We need to use a UMD module to build the static site.
+      libraryTarget: "umd",
+      globalObject: "this",
+    },
+    resolve: {
+      // This allows you to set a fallback for where Webpack should look for modules.
+      // We placed these paths second because we want `node_modules` to "win"
+      // if there are any conflicts. This matches Node resolution mechanism.
+      // https://github.com/facebookincubator/create-react-app/issues/253
+      modules: [
+        "node_modules",
+        paths.appNodeModules,
+        ...(process.env.NODE_PATH || "").split(path.delimiter).filter(Boolean),
+      ],
+      // These are the reasonable defaults supported by the Node ecosystem.
+      // We also include JSX as a common component filename extension to support
+      // some tools, although we do not recommend using it, see:
+      // https://github.com/facebookincubator/create-react-app/issues/290
+      // `web` extension prefixes have been added for better support
+      // for React Native Web.
+      extensions: [".web.js", ".mjs", ".js", ".json", ".web.jsx", ".jsx"],
+      alias: {
+        // Support React Native Web
+        // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
+        "react-native": "react-native-web",
```

Comment:
whoa where did rn come from. RN is my life lol

Reply:
I'm not sure, tbh. Note this is a copied file. :)